docs/usage/shoot_settings/shoot_networking.md

```diff
@@ -9,9 +10,6 @@ This document contains network related information for Shoot clusters.
 
 ## Pod Network
 
-A Pod network is imperative for any kind of cluster communication with Pods not started within the Node's host network.
```

Comment:
What is the motivation for the changes in this file and what we improve with them?

Reply:
The text explanation and the examples to be grouped together. Also the above sentence was removed because it does not provide any value